calour/experiment.py

```diff
@@ -15,6 +15,8 @@
 import scipy
 import biom
 
+import calour as ca
+
 
```

Comment:
could you remove the changes of `experiment.py` for now? It will conflict with my changes.